docs/pages/more/expo-cli.mdx

```diff
@@ -251,6 +252,82 @@ The following options are provided:
 - `-c, --clear`: Clear the bundler cache before exporting.
 - `--no-minify`: Skip minifying JavaScript and CSS assets (SDK 49 and above).
 
+#### Hosting with sub-paths
+
+> Experimental functionality, first available in the upcoming SDK 50.
+
+<Tabs>
+
+<Tab label="SDK 49 and below">
+
+    Exporting for a sub-path is not supported in SDK 49 and below.
+
+</Tab>
+
+<Tab label="SDK 50 and above">
+
+You can configure the prefix for static assets by setting the `experiments.basePath` field in your `app.json`:
+
+```json app.json
+{
+  "expo": {
+    "experiments": {
+      "basePath": "/my-root"
+    }
+  }
+}
+```
+
+This will export the website with all resources prefixed with `/my-root`. For example, an image at `assets/image.png` will be expected to be hosted at `/my-root/assets/image.png`. The actual file will be located in the same file system location as before as the entire directory is expected to be hosted at `/my-root` on the server.
```

Comment:
```suggestion
This will export the website with all resources prefixed with `/my-root`. For example, an image at `assets/image.png` will be expected to be hosted at **/my-root/assets/image.png**. The actual file will be located in the same file system location as the entire directory is expected to be hosted at `/my-root` on the server.
```

- Didn't understand the context behind what was "before".
- [File/directory names to be bold as per our writing styleguide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#filenames-as-bold-text)